apl/tabular-operators/lookup-operator.mdx

```diff
@@ -0,0 +1,132 @@
+    - `inner` lookup only includes rows from the primary dataset if they match the conditions. Unmatched rows are excluded from the output.
+- `Conditions`: The conditions for matching rows from `PrimaryDataset` to rows from `LookupTable`. The conditions are equality expressions that determine how Axiom matches rows from the `PrimaryDataset` (left side of the equality expression) with rows from the `LookupTable` (right side of the equality expression). The two sides of the equality expression must have the same data type.
+    - To use `lookup` on a key column that has the same name in the two datasets, simply use the field name. For example, `on id`.
+    - To use `lookup` on a key column that has different names in the two datasets, define the two field names in an equality expression such as `on id == trace_id`.
+    - You can define multiple conditions. To separate conditions, use commas (`,`). Don’t use `and`. For example, `on id == trace_id, span == span_id`.
+
+### Returns
+
+A dataset where rows from `PrimaryDataset` are enriched with matching columns from `LookupTable` based on the key column.
+
+## Use case example
+
+Add a field with human-readable names for each service.
+
+**Query**
+
+```kusto
+let LookupTable=datatable(serviceName:string, humanreadableServiceName:string)[
+	'frontend', 'Frontend',
+	'frontendproxy', 'Frontend proxy',
+	'flagd', 'Flagd',
+	'productcatalogservice', 'Product catalog',
+	'loadgenerator', 'Load generator',
+	'checkoutservice', 'Checkout',
+	'cartservice', 'Cart',
+	'recommendationservice', 'Recommendations',
+	'emailservice', 'Email',
+	'adservice', 'Ads',
+	'shippingservice', 'Shipping',
+	'quoteservice', 'Quote',
+	'currencyservice', 'Currency',
+	'paymentservice', 'Payment',
+	'frauddetectionservice', 'Fraud detection',
+];
+['otel-demo-traces']
+| lookup kind=leftouter LookupTable on $left.['service.name'] == $right.serviceName
+```
+
+[Run in Playground](https://play.axiom.co/axiom-play-qf1k/query?initForm=%7B%22apl%22%3A%22let%20LookupTable%3Ddatatable(serviceName%3Astring%2C%20humanreadableServiceName%3Astring)%5B%20'frontend'%2C%20'Frontend'%2C%20'frontendproxy'%2C%20'Frontend%20proxy'%2C%20'flagd'%2C%20'Flagd'%2C%20'productcatalogservice'%2C%20'Product%20catalog'%2C%20'loadgenerator'%2C%20'Load%20generator'%2C%20'checkoutservice'%2C%20'Checkout'%2C%20'cartservice'%2C%20'Cart'%2C%20'recommendationservice'%2C%20'Recommendations'%2C%20'emailservice'%2C%20'Email'%2C%20'adservice'%2C%20'Ads'%2C%20'shippingservice'%2C%20'Shipping'%2C%20'quoteservice'%2C%20'Quote'%2C%20'currencyservice'%2C%20'Currency'%2C%20'paymentservice'%2C%20'Payment'%2C%20'frauddetectionservice'%2C%20'Fraud%20detection'%2C%20%5D%3B%20%5B'otel-demo-traces'%5D%20%7C%20lookup%20kind%3Dleftouter%20LookupTable%20on%20%24left.%5B'service.name'%5D%20%3D%3D%20%24right.serviceName%22%7D)
```

Comment:
I would add a ` | project _time, span_id, ['service.name'], humanreadableServiceName` at the end so that it matches the output. Also our experience sucks with so many fields.